crates/adapters/src/integrated/delta_table/output.rs

```diff
@@ -239,11 +235,22 @@ impl WriterTask {
         // and hope for the best.  Without this config option, writes to S3-based delta tables will fail.
         storage_options.insert("AWS_S3_ALLOW_UNSAFE_RENAME".to_string(), "true".to_string());
 
-        let delta_table = CreateBuilder::new()
-            .with_location(inner.config.uri.clone())
+        let save_mode = match inner.config.mode {
             // I expected `SaveMode::Append` to be the correct setting, but
             // that always returns an error.
-            .with_save_mode(SaveMode::Ignore)
+            DeltaTableWriteMode::Append => SaveMode::Ignore,
```

Comment:
Who designed this api?

Reply:
yeah, this one is completely broken: `Ignore` is really `Append`, `Append` doesn't work, and `Overwrite` doesn't overwrite anything...